recipe/meta.yaml

```diff
@@ -35,8 +32,7 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/lib/libglfw.so               # [linux]
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
```

Comment:
Sure you _wanted_ to remove this line too?

Reply:
Yeah I think cons build does this automatically now